radish34/docker-compose.yml

```diff
@@ -113,13 +100,13 @@ services:
     command: npm run dev
     restart: on-failure
     volumes:
+      - ./api/src:/app/src:delegated
```

Comment:
any reason to map src as well? dist should contain all the compiled js files upon build/restart

Reply:
I thought this at first too. Since the babel compiler/watcher is running inside the container and watching the `src` directory for changes, it needs to have access to those files. Otherwise, it will not pickup on any file changes in that directory. I tried removing `src` from the mounted directories. It causes an error message `babel: src does not exist` and does not recompile after making changes to the local `src` files.